### ISSUE TITLE
Needs to be more descriptive 

### ISSUE THREAD

**Issue by darksuei** [qgustavor/megajs-docs#5]
Hello, I recently used Mega JS to create a storage application. Source code -> https://github.com/darksuei/cloudwave , and honestly the documentation was quite hard to follow. Had to view the mega js source code lots of times to see how things really worked.

I'm gonna attempt to work on this documentation and send a PR soon. This is just to check in that my work wouldn't be a waste of time and effort. Do let me know if I could carry on. @qgustavor 👏

**Comment by qgustavor** [comment]
Ok, you can send a PR. As you can see from how I've been handling the main repository I'm a bit burned up. I guess that's because I'm almost not using this library anymore. There are just two projects of mine still using MEGAJS but both are unmaintained. I just keep then running but the day those stop working I will not fix any issues on them.

I agree the documentation is not great, I spent a lot of time writing it but I know that's not great. Last time I used this library I had to use the documentation because *not even I* remembered how to use it, but I noticed some issues (which led to 6a8dc1c1db5554c29de6e65aaecbf0cf7b994056 and 5d914e77d3604f3360b371958cd42f6b7085dbae).

I'm not willing to transfer control of this project to anyone else soon because I remember it happening with some NPM libraries badly. If someone wants to continue this project I think it would be someone willing to make a fork and improving upon it, if someone does that and I see that it's a good maintainer - not someone wanting to monetize the project or add malware to it - I think I can transfer control of this project to them.

Just send me PRs, I'll review then and approve. Don't need to open new issues.

**Comment by darksuei** [comment]
Okay, Ill start working on the docs. Is there a way I can reach out to you though?

**Comment by qgustavor** [comment]
It can be here, but just send a PR, no?
